src/main/scala/is/hail/expr/Relational.scala

```diff
@@ -1536,12 +1538,12 @@ case class TableImport(paths: Array[String], typ: TableType, readerOpts: TableRe
 
   def copy(newChildren: IndexedSeq[BaseIR]): TableImport = {
     assert(newChildren.isEmpty)
-    this
+    TableImport(paths, typ, readerOpts)
   }
 
   def execute(hc: HailContext): TableValue = {
     val rowTyp = typ.rowType
-    val nFieldOrig = readerOpts.originalRowTypeSize
+    val nFieldOrig = readerOpts.originalType.size
```

Comment:
Changing import is probably best as a second PR.

Reply:
now that I have it, is it worth ripping out? I added it in the first place to get better integration tests to help with debugging.